src/panels/ActionsPanel/ActionsPanel.test.tsx

```diff
@@ -19,6 +19,7 @@ import {
 import { renderComponent } from "testing/utils";
 
 import ActionsPanel from "./ActionsPanel";
+import { Label as ConfirmationDialogLabel } from "./ConfirmationDialog/types";
```

Comment:
This should be exported as `ConfirmationDialogLabel` from "./ConfirmationDialog/index.ts"